docs/roadmap.md

```diff
@@ -4,6 +4,149 @@ title: Babel Roadmap
 sidebar_label: Roadmap
 ---
 
+This document outlines some of the improvements our team members would like to work on during this year.
+
+This is far from being a complete list of all the new features or important changes that we'll bring to Babel, but it's a good summary if you are interested in the general direction that the project is moving to. We might actually not do every listed point, or we might delay some of them to the next year. Some of them have clear starting and ending points, while others need more research or [RFCs](https://github.com/babel/rfcs).
+
+If your company is interested in some of these and would like to directly sponsor their implementation please [reach out](mail:team@babeljs.io)!
+
+## Babel 2021 Roadmap
+
+### Babel 8
+
+We have been talking about the Babel 8 release for more than one year (actually, we initially scheduled it for about one year ago)! However, we are now closer then ever to actually release it.
+
+Most of the remaining tasks are in the [tracking issue](https://github.com/babel/babel/issues/10746), but there are still a few blockers:
+- We want to drop support for [Node.js 10](https://github.com/nodejs/Release), which stops being maintained on 2021-04-30;
+- We would like to release Babel as a pure ESM package. We are now in the process of converting our sources to be compatible with Node.js' ESM implementation, and while doing it we are examining how we can make it easier for people that currently use Babel to compile ESM to CJS;
+- We are trying to align our TypeScript AST with the `typescript-eslint` project. Our ASTs are _almost_ identical, but we need to introduce some small breaking changes to fully align;
+- Our release infrastructure doesn't support yet pre-releases, or using multiple "main" branches (one for Babel 8 and one for Babel 7);
+- We haven't figured out yet a policy for Babel 7 maintainance.
```

Comment:
```suggestion
- We haven't figured out yet a policy for Babel 7 maintenance.
```

Reply:
50% of the times I search on google how to spell it, the other 50% I spell it wrong :joy: